src/crimson/os/Finisher.cc

```diff
@@ -0,0 +1,97 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
```

Comment:
I guess this might be pretty duplicative. Is `#ifdef WITH_SEASTAR` a viable option for e.g. `Finisher.cc` or `MemStore.{cc,h}`?